### ISSUE TITLE
Add feature page

### ISSUE THREAD

**Issue by soc** [scala-android/scala-android.github.io#18]
sbt-android has so many amazing features, I think it would be great if we had a dedicated page demonstrating them!

I'm thinking of
- SDK management
- Typed resources and friends
- ProGuard caching
- Protify

Anything else we should mention?

**Comment by pfn** [comment]
Indeed, it would also be nice to showcase the full list of device management commands:
- `pidcat`, `pidcat-grep`
- `logcat`, `logcat-grep`
- `adb-shell`, `adb-reboot`, `adb-screenon`, `adb-runas`, `adb-kill`, `adb-wifi`
- `adb-ls`, `adb-cat`, `adb-rm`, `adb-push`, `adb-pull` (all with full tab completion for remote and local files)
- `devices`
